hazelcast-jet-sql/src/main/java/com/hazelcast/jet/sql/impl/inject/JsonUpsertTarget.java

```diff
@@ -16,25 +16,26 @@
 
 package com.hazelcast.jet.sql.impl.inject;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
 import com.hazelcast.sql.impl.QueryException;
 import com.hazelcast.sql.impl.type.QueryDataType;
 
 import javax.annotation.Nullable;
 import javax.annotation.concurrent.NotThreadSafe;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
 
 import static com.hazelcast.jet.impl.util.ExceptionUtil.sneakyThrow;
 import static com.hazelcast.jet.sql.impl.inject.UpsertInjector.FAILING_TOP_LEVEL_INJECTOR;
 
 @NotThreadSafe
 class JsonUpsertTarget implements UpsertTarget {
 
-    private static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
`com.fasterxml.jackson.core:jackson-databind` isn't needed anymore?

Reply:
It is, when reading. I only adapted writing.

Reply:
after the last commit it's still needed when writing: we support OBJECT fields of type `JsonNode`, which is from jackson-databind